SparkPost/2web2ui
# FE-1022 - Hide month pagination buttons with CSS instead of conditional rendering

[FE-1022](https://jira.int.messagesystems.com/browse/FE-1022)

### What Changed
- Updates `<Navbar/>` component in `DateSelector.js` to conditionally show and hide components instead of using conditional rendering
- Adds `type="button"` to next and previous buttons in case this component is ever used inside a `<form>`
- Updates unit tests to match new implementation

### How To Test
- Go through signals pages and open the date picker - verify that clicking the previous button and then the next button does not close the popover
- Verify this fix works both for the OG theme and the Hibana theme

### To Do
